src/types/StateSpace.jl

```diff
@@ -53,7 +53,7 @@ end
 """ If D=0 then convert to correct size and type, else, create 1x1 matrix"""
```

Comment:
```suggestion
""" If D=0 then create zero matrix of correct size and type, else, convert D to correct size and type"""
```

Reply:
```suggestion
""" If D=0 then create zero matrix of correct size and type, else, convert D to correct type"""
```
I agree. Sounds a bit off that the size of D should be converted in that case, though.

Reply:
```suggestion
""" If D=0 then create zero matrix of correct size and type, else, convert D to correct shape and type"""
```
Is what I intended to write, since `to_matrix` might to a reshape if D is a vector. But either is fine for me.